src/codeu/chat/common/BasicView.java

```diff
@@ -51,5 +52,7 @@
   //
   //   Return all messages whose id is found in the given collection.
   Collection<Message> getMessages(Collection<Uuid> ids);
+    
+  ServerInfo getInfo();    
```

Comment:
I don't think this method is actually called by your code. There's already a getInfo method in Context.java that's already used. You should remove this method since it's unused.